content/blog/20181125/index.md

```diff
@@ -19,6 +19,6 @@ type: tour
 wiki: https://pl.wikipedia.org/wiki/Klimczok
 ---
 
-One of the best photos I ever made in the mountains. There was a fog in the city, but after climbing a bit I get an awesome inversion.
```

Comment:
@msapa actually inversion is a think https://en.wikipedia.org/wiki/Inversion_(meteorology)